app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt

```diff
@@ -101,16 +101,18 @@ fun PatchesSelectorScreen(
             )
         },
         floatingActionButton = {
-            ExtendedFloatingActionButton(
-                text = { Text(stringResource(R.string.patch)) },
-                icon = { Icon(Icons.Default.Build, null) },
-                onClick = {
-                    composableScope.launch {
-                        // TODO: only allow this if all required options have been set.
-                        onPatchClick(vm.getAndSaveSelection(), vm.getOptions())
+            if (!vm.isSelectionEmpty()) {
+                ExtendedFloatingActionButton(
```

Comment:
I believe it makes more sense to disable the button instead of hiding it completely

Reply:
Do you know how to disable `ExtendedFloatingActionButton`? I can't seem to find how.

Reply:
I don't think that the `EntendedFloatingActionButton` comes with a disable param.

Reply:
Hm, maybe this helps? you'll need to make a few changes but nothing crazy

https://github.com/ReVanced/revanced-manager-compose-old/blob/main/app/src/main/java/app/revanced/manager/ui/component/FloatingActionButton.kt

Reply:
That is helpful! Thanks!

Reply:
ExtendedExtendedFloatingButton cooking up 🔥